extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -88,6 +105,11 @@ export class PodcastPlayer extends Component {
 	 */
 	handleError = () => {
 		this.setState( { playerState: STATE_ERROR } );
+
+		speak(
+			`${ sprintf( __( 'Error: Episode unavailable. (Open in a new tab)', 'jetpack' ) ) }`,
```

Comment:
Do you need `sprintf` here since you don't have a placeholder?

Reply:
Maybe we can use a `sprintf` construct in https://github.com/Automattic/jetpack/pull/15207#discussion_r401439353 ?

Reply:
I originally had a `title` being used by `sprintf()` but forgot to remove it after I removed the `title`. Good catch!

Reply:
Removed 👍